source/development/wazuh-logtest.rst

```diff
@@ -0,0 +1,63 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. _dev-wazuh-logtest:
+
+
+Wazuh Logtest
+=============
+
+The Wazuh-Logtest solution is distributed by different components of the Wazuh environment:
+
+**Core Logtest**
+
+    The logtest core is part of the *Analysis daemon* (in manager). It has a session manager and shares its rules engine with Logtest.
+
+**Wazuh RESTful API**
+
+    Interact with Core Logtest, and provide an interface to test the rules and decoders remotely.
```

Comment:
Change `test the rules and decoders` to `test rules and decoders` or `test the ruleset`
Also `provide a remote interface` is more accurate